### ISSUE TITLE
package customization

### ISSUE THREAD

**Issue by PalmerAL** [nt1m/material-framework#87]
something like http://modernizr.com/download/

**Comment by nt1m** [comment]
That would require splitting up the current CSS into multiple file, and use a task runner to put the whole thing together.

**Comment by PalmerAL** [comment]
Right. The current CSS should probably be split into multiple files at some point anyway, though.
